src/validation/fields/date.validation.ts

```diff
@@ -92,45 +91,12 @@ const is_trust_still_active_validation = (error_message: string) => [
     )),
 ];
 
-const checkAgainstFilingDate = (date_field_id, error_message) =>
```

Comment:
Could these functions not remain where they are and only the `checkDatePreviousToFilingDate` moves, as that's the one that makes the async call?

Reply:
No, because all these functions have been transformed because they use an async function and so must be async.